packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-expressions */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  afterEach: (browser, done) => {
```

Comment:
I don't know why the build is still passing without these changes, but if you inspect the output of the travis CI build, it's throwing a bunch of errors near the bottom ( I think this branch doesn't have the changes to the button ). These tests will throw the same errors without these changes.

( Copy pasted from above )

I don't know if the generator is updated to include some of the recent changes to testing harness Ryan implemented. The syntax for these scripts has changed a little bit.

https://github.com/cerner/terra-core/blob/master/packages/terra-button/package.json#L37

Here are pull request implementing the differences: 
https://github.com/cerner/terra-core/pull/180
https://github.com/cerner/terra-core/pull/167